addons/sale/sale.py

```diff
@@ -824,6 +824,18 @@ def confirm_paid(self):
             order.message_post(body=_("Invoice %s paid") % (name))
         return res
 
+    @api.model
+    def _refund_cleanup_lines(self, lines):
+        result = super(AccountInvoice, self)._refund_cleanup_lines(lines=lines)
+        for line in lines:
+            values = {}
+            for name, field in line._fields.iteritems():
```

Comment:
Why use iteritems() here?  You haven't used `field` in loop

Second thing loop is not required here you can directly write 
`values['sale_line_ids'] = [(6, 0, line['sale_line_ids'].ids)]`